python/sglang/srt/models/deepseek_v2.py

```diff
@@ -573,13 +576,20 @@ def forward_absorb(
             )
         q_nope, q_pe = q.split([self.qk_nope_head_dim, self.qk_rope_head_dim], dim=-1)
 
-        if self.w_kc.dtype == torch.float8_e4m3fn:
-            q_nope_val, q_nope_scale = input_to_float8(
-                q_nope.transpose(0, 1), torch.float8_e4m3fn
-            )
-            q_nope_out = bmm_fp8(
-                q_nope_val, self.w_kc, q_nope_scale, self.w_scale, torch.bfloat16
-            )
+        if self.w_kc.dtype == torch.float8_e4m3fn or torch.float8_e4m3fnuz:
+            if is_hip_:
+                # TODO(flashinfer): add bmm_fp8 for torch.float8_e4m3fnuz
```

Comment:
Please remove this comment. I believe FlashInfer doesn't plan to support AMD bmm_fp8.

Reply:
@zhyncs we will do that, so please be ok.

Reply:
You can directly add support in `sgl-kernel`.

Reply:
@zhyncs we can do both, but for the comment, it is just that not a decision or roadmap 😄